tutorials/MD/archive-2019/CONTRIBUTORS.md

```diff
@@ -0,0 +1,9 @@
+# List of contributors to the Oxford Computational Biochemitry Molecular Dynamics course
```

Comment:
I think can we can ahead and just delete all the archive-2019 stuff. It's already in the 2019 release, so there's a git memory of it.

Reply:
Yes - please delete.

Reply:
Deleted.